src/service/mpris.ts

```diff
@@ -197,7 +198,7 @@ class MprisPlayer extends Service {
             Gio.FileCopyFlags.OVERWRITE,
             GLib.PRIORITY_DEFAULT,
             // @ts-expect-error
```

Comment:
Thanks for this PR! I'm trying to learn this framework, and these will help significantly! Found a small issue:

```suggestion
```

This line causes an error because the type was fixed, but uses a comment saying it's wrong. Removing it fixes the build.